src/gkeClient.ts

```diff
@@ -112,7 +112,7 @@ export class ClusterClient {
     const matches = name.match(membershipResourceNamePattern);
     if (!matches) {
       throw new Error(
-        `Failed to parse membership name "${name}": invalid pattern. Should be of form projects/PROJECT_ID/locations/LOCATION/memberships/MEMBERSHIP_NAME`,
+        `Failed to parse membership name "${name}": invalid pattern. Should be of form projects/PROJECT_ID/locations/LOCATION/gkeMemberships/MEMBERSHIP_NAME or projects/PROJECT_ID/locations/LOCATION/memberships/MEMBERSHIP_NAME`,
```

Comment:
Per b/261052807, wouldn't memberships.Get always return in format `projects/p/locations/l/memberships/m`? My understanding was only the connect gateway URL supports multiple collection types.

Reply:
Theoretically yes, but I want to handle the case where someone copy-pastes from their kubeconfig, since this is extremely nuanced.